.github/workflows/backstage-release.yaml

```diff
@@ -0,0 +1,72 @@
+name: Publish release on NPM
+
+on:
+  # When manually triggered
+  workflow_dispatch:
+  # When there's a release created
+  release:
+    types: [ published ]
+
+jobs:
+  publish-release-on-npm:
+    name: Publish release on NPM
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: 'backstage/.nvmrc'
+          cache: 'yarn'
+          cache-dependency-path: 'backstage/yarn.lock'
+
+      - name: Print environment
+        run: |
+          node --version
+          yarn --version
+
+      - name: Print release name
+        run: |
+          echo "Release name: ${{ github.event.release.name }}"
```

Comment:
I think this variable is not present when this is triggered on `workflow_dispatch`

Reply:
Good point

Reply:
Should be good now